packages/fiori-freestyle-writer/jest.config.js

```diff
@@ -3,7 +3,16 @@ module.exports = {
         '^.+\\.ts$': 'ts-jest'
     },
     collectCoverage: true,
-    coverageReporters: ['text','html'],
     collectCoverageFrom: ['src/**/*.ts'],
+    reporters: [
+        'default',
```

Comment:
I assume `default`  covers  `coverageReporters: ['text','html'],` ?

Reply:
@devinea: I couldn't find docs on the default reporters of jest. But, I did verify that the coverage is shown in the console and the html files are in the `coverage/lcov-report` folder.